app/services/publish_manual_service.rb

```diff
@@ -8,6 +10,8 @@ def initialize(dependencies)
 
   def call
     if versions_match?
+      tags
```

Comment:
`update_manual_with_tags` already fetches the tags, so we shouldn't need to call `tags` separately here.